getting_started/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -151,13 +151,16 @@ as with any project. Many IDEs (such as vs code) can add packages directly. They
 can also be added manually by adding the package reference in the .csproj file
 located in the project root:
 
-      <ItemGroup>
-        <PackageReference Include="Newtonsoft.Json" Version="11.0.2"/>
-      </ItemGroup>
+.. code-block:: xml
+   :emphasize-lines: 2
+
+        <ItemGroup>
+            <PackageReference Include="Newtonsoft.Json" Version="11.0.2"/>
+        </ItemGroup>
       ...
     </Project>
```

Comment:
Is this correct? Indentation looks suspect witht hat dangling `</Project>`, but didn't test/build to check

Reply:
This is probably to give an idea to where this part of the code is located.